vm/vm_jit.go

```diff
@@ -1,31 +1,270 @@
+// +build evmjit
+
 package vm
 
 import "math/big"
+import "github.com/ethereum/go-ethereum/crypto"
+import "github.com/ethereum/go-ethereum/state"
+
+/*
+#include "../evmjit/libevmjit/interface.h"
+
+#cgo LDFLAGS: -L/home/chfast/go/src/github.com/ethereum/go-ethereum/evmjit/build/libevmjit -levmjit
```

Comment:
I think it would better to just depend on an installed version (maybe that's what you meant).

There is no way to integrate cmake with `go build`. The `go` tool is intentionally stupid because
abominations of good taste such as `cmake` have no place in the beautifully simple world of Go.